ipa-core/src/protocol/dp/insecure.rs

```diff
@@ -101,6 +101,63 @@ impl DiscreteDp {
     }
 }
 
+///  Non-negative DP noise for OPRF padding
+///  Samples from a Truncated Double Geometric
+#[derive(Debug)]
+pub struct OPRFPaddingDp {
+    epsilon: f64,
+    delta: f64,
+    truncated_double_geometric: TruncatedDoubleGeometric,
+}
+
+fn right_hand_side(n: i32, big_delta: i32, epsilon: f64) -> f64 {
```

Comment:
same here, some documentation would be great to have here

Reply:
added documentation here and in dp/README.md